bigchaindb/__init__.py

```diff
@@ -99,13 +59,12 @@
         'advertised_host': 'localhost',
         'advertised_port': 9985,
     },
+    'tendermint': {
+        'host': 'localhost',
+        'port': 46657,
```

Comment:
These two new BigchainDB configuration settings should also be documented in the server docs page about configuration settings: `docs/server/source/server-reference/configuration.md`

Reply:
As discussed, I am removing changes from the PR and will address in another one.